DoctorWho.Domain/EpisodeCompanion.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DoctorWho.Domain
+{
+    public class EpisodeCompanion
+    {
+        public int EpisodeCompanionId { get; set; } 
+        public int EpisodeId { get; set; } 
+        public int CompanionId { get; set; }
+
+        [NotMapped]
+        public Episode Episode { get; set; }
+
```

Comment:
I think EF core will discover  that this is a navigational property without the attribute